backend/src/jobs/updateMonthlyBalance.ts

```diff
@@ -0,0 +1,70 @@
+const dotenv = require("dotenv");
+const path = require("path");
+dotenv.config({ path: path.resolve(__dirname, "../../../.env") });
+const { createClient } = require("@supabase/supabase-js");
+const supabase = createClient(
```

Comment:
Database related functions can be inside of 
[backend](https://github.com/Winxton/SubShare/tree/main/backend)/[src](https://github.com/Winxton/SubShare/tree/main/backend/src)
/repository/

since the client has already been created there.